src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
Remove React import and only import what you need to be consistent with rest of application.